### DISCUSSION TITLE
Clipping a netcdf file with Shapefile in Anaconda produces NaN values

### DISCUSSION THREAD

**Discussion by MayMonThetKhine** [pydata/xarray#6572] [Q&A] [answered]
I want to clip my netcdf file with the shapefile, the script I run is as below. But my clipped data when I check with Panoply gives the dtr values all NaN. I attached the screenshot here.

Please enlighten me on how I can solve this one and where I went wrong. Thanks much.

==================================================================================

```python
import xarray
import rioxarray
import geopandas
from shapely.geometry import mapping

xds = xarray.open_dataset("dtr_MON_climpact.sample_historical_NA_1970-1989.nc", drop_variables='time_bnds')
xds.rio.write_crs("EPSG:4326", inplace=True)
geodf = geopandas.read_file("Chauk_BD.shx")

clipped = xds.rio.clip(geodf.geometry.apply(mapping), geodf.crs,drop=False)
clipped.to_netcdf('my clipped data 1.nc')
```

![NaN values](https://user-images.githubusercontent.com/104421747/166858383-ca8db1e0-b124-4b3f-a8b1-5313789638a3.png)

**Comment by andersy005** [comment]
@MMon27, can please use the rioxarray discussion forum: https://github.com/corteva/rioxarray/discussions ?

**Reply by MayMonThetKhine** [reply]
Thank you for providing the link. I will do so.

**Reply by andersy005** [reply]
thank you, @MMon27!

**Comment by andersy005** [comment, marked as answer]
For anyone interested in this topic, this discussion has been moved to https://github.com/corteva/rioxarray/discussions/520
